addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -1222,6 +1247,53 @@ options.registry.AddField = FieldEditor.extend({
     },
 });
 
+options.registry.WebsiteFormSubmit = options.Class.extend({
+    /**
+     * Select the submit button's position
+     */
+    selectButtonPosition: function (previewMode, value, params) {
+        this.$target[0].querySelectorAll('.s_website_form_label').forEach(el => el.remove());
+        this.$target[0].classList.remove('text-center');
+        const button = this.$target[0].querySelector('.s_website_form_send');
+        button.classList.remove('float-right', 'float-left');
+        switch (value) {
+            case 'center':
+                this.$target[0].classList.add('text-center');
+                break;
+            case 'align': {
+                const div = document.createElement('div');
+                const formOptions = this.el.closest('.o_we_customize_panel');
+                div.style.width = formOptions && formOptions.querySelector('we-customizeblock-options [data-select-style] input').value + 'px',
+                div.classList = 's_website_form_label';
+                this.$target[0].insertBefore(div, button);
+                break;
+            }
+            default:
+                button.classList.add(`float-${value}`);
+                break;
+        }
+    },
```

Comment:
Did not see that in fact, don't use floating stuff, it works kinda by chance. If the row was not using flex, end a form with a floating button would make overflow over the next snippet. In fact it is probably the case in some browser, I cannot find the spec that says floating stuff are cleared when combined with flex.

The option should toggle the text-left/text-right-text-center on the parent col (in fact, I wonder why you do it for center and not the other?)

Reply:
Never used `text-left` and `text-right`... Now I know !